ast/shared/src/main/scala/jawn/ast/JValue.scala

```diff
@@ -81,6 +82,7 @@ sealed abstract class JValue {
 
 object JValue {
   implicit val facade: Facade[JValue] = JawnFacade
+  implicit val classTag: ClassTag[JValue] = ClassTag(classOf[JValue])
```

Comment:
nit: I wonder if it's worth making this `lazy`?

Reply:
What are you trying to optimize? The memory consumption?
In the other hand the `UnwrapArray` parser would have to repeatedly go through the synchronized access.

Reply:
Load time. Companion object gets initialized the moment the type gets used so I thought it might be worth it to defer reflection call until the code path hits it.

Reply:
Sounds reasonable :+1: 

I made it lazy.